src/Razor/src/Microsoft.AspNetCore.Razor.Language/Intermediate/HtmlContentIntermediateNodeExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Razor.Language.Intermediate
+{
+    public static class HtmlContentIntermediateNodeExtensions
```

Comment:
Can you just make these members of the class? There's no reason for these to be extensions.